Makefile

```diff
@@ -1,11 +1,26 @@
+export GOPROXY=https://proxy.golang.org
+
+GO := go
+GO110 := 1.10
```

Comment:
should this be bumped too?

Reply:
Hmm, don't think we actually need that.  Removing it.